lib/autoupdate/agent/setup.go

```diff
@@ -258,7 +316,11 @@ func writeTemplate(path, t string, values any) error {
 	}
 	defer f.Cleanup()
 
-	tmpl, err := template.New(file).Parse(t)
+	tmpl, err := template.New(file).Funcs(template.FuncMap{
+		"replace": func(s, old, new string) string {
+			return strings.Replace(s, old, new, -1)
```

Comment:
nit: `strings.ReplaceAll` little bit simpler to read, so you don't need to go to read godoc